cdap-unit-test/src/test/java/co/cask/cdap/spark/metrics/SparkMetricsIntegrationTestRun.java

```diff
@@ -44,7 +45,7 @@
 @Category(XSlowTests.class)
 public class SparkMetricsIntegrationTestRun extends TestFrameworkTestBase {
 
-  public static final String METRICS_KEY = "system.driver.BlockManager.memory.remainingMem_MB";
+  private static final String METRICS_KEY = "system.<driver>.BlockManager.memory.remainingMem_MB";
```

Comment:
don't normalize. We would have to do that for many other things, and many versions. Unless we want to map this to a fixed documented CDAP metric. In that case I would keep the original plus the CDAP one.